app/controllers/comments_controller.rb

```diff
@@ -62,6 +63,13 @@ def destroy
     def set_comment
       @comment = Comment.find(params[:id])
     end
+    
+    def is_an_authorized_user
```

Comment:
The logic for checking user authorization is straightforward and clear. Great Job Maya!!